libraries/botbuilder-dialogs-adaptive/src/actions/baseInvokeDialog.ts

```diff
@@ -72,7 +73,12 @@ export class BaseInvokeDialog<O extends object = {}> extends Dialog<O> implement
 
         for (const key in bindingOptions) {
             const bindingValue = bindingOptions[key];
-            const value = new ValueExpression(bindingValue).getValue(dc.state);
+            let value = new ValueExpression(bindingValue).getValue(dc.state);
+
+            if (value) {
+                value = replaceJsonRecursively(dc.state, value);
+            }
```

Comment:
This does not look safe to me at all--are you really sure that no child is a string that should not be replaced?